spec/oas3.patched/openapi/public/components/schemas/ProjectOrganization.yaml

```diff
@@ -11,5 +11,5 @@ properties:
     format: date-time
     type: string
   href:
-    $ref: './Href.yaml'
+    type: string
```

Comment:
Given that this is adjusting an existing patch, I think it's OK to merge as-is, but we will likely do something slightly different upstream, and make `ProjectOrganization` a `oneOf` between the `Href` and `Organization` models; that better supports the ability to bring back additional organization attributes via `include` parameters, and if the API is returning an organization ID, then the organization href is not that critical (in the SDK, it's easier to load data with an object ID than with an href, at least for now).